ansible/roles/provision_app/tasks/main.yml

```diff
@@ -18,29 +18,25 @@
 # - include_role: name=install_app
 #   when: deployment_environment_id in ['test', 'stable']
 
-- include: build.yml
-  when: deployment_environment_id != 'production'
+- include: service.yml
 
+- block:
+  - include: build.yml
 
-- name: Check if we can find the schema in our source code repo
-  stat:
-    path: "{{ app.source }}/{{ app.sqlschema }}"
-  register: sqlschema_file
+  - name: Check if we can find the schema in our source code repo
+    stat:
+      path: "{{ app.source }}/{{ app.sqlschema }}"
```

Comment:
here we have `app.source` instead of `app.base`

Reply:
It is annoying but the difference is intentional, since the location and availability of schema.sql (which originates from qvain-api) depends on whether we have the qvain-api source or the built files present. The issue is again that we have two different types of provisioning:
1) Provision, download source and build on target machine in one command (local_development)
2) Provision target machine, provide built files separately with install_app role (production)

With (1) the qvain-api source contains schema.sql, with (2) schema.sql comes with install_app. 

(A bit off-topic: stable and test are a currently combination of these provisioning types since they are provisioned with (1) so they build qvain themselves at first, but in practice they are updated with install_app. I'd rather have them be like the production config where they don't have git or build tools. I'm also not sure we ever really need to build automatically on the target machine.)